src/main/java/com/endava/soa_5ed/task1_basics/Ex3.java

```diff
@@ -0,0 +1,39 @@
+package com.endava.soa_5ed.task1_basics;
+
+
+public class Ex3 {
+    //    Write a method that prints day of the week by given number. Use switch case!
+
+    String printDay(int number){
+        String day;
+        switch(number){
+            case 1:
+                day = "Monday";
+                break;
+            case 2:
+                day = "Tuesday";
+                break;
+            case 3:
+                day = "Wednesday";
+                break;
+            case 4:
+                day = "Thursday";
+                break;
+            case 5:
+                day = "Friday";
+                break;
+            case 6:
+                day = "Saturday";
+                break;
+            case 7:
+                day = "Sunday";
+                break;
+            default:
+                day = "";
+                System.out.println("The provided number is invalid");
```

Comment:
a good practice is to return the number which is invalid so the used would know what number entered this switch-case loop. It makes easier to debug if there is an unexpected issue

Reply:
done.